src/Pages/Events/EventModal.tsx

```diff
@@ -78,10 +80,14 @@ const EventModal = ({
               placeholder=""
             />
           </div>
-          <div className="col-md-4">
+          <div className="col-md-4" title="Event repeats every __ weeks">
```

Comment:
Não faças as tooltips assim. Usa como já se faz noutros sítios. No ficheiro `DashboardContent.tsx` até tens um exemplo muito simples de exemplo.

Neste caso, ficava do estilo:
```suggestion
<div className="col-md-4">
  <span className="text-dark small text-uppercase">
    <strong>Periodic</strong>
  </span>
  <FontAwesomeIcon
    className="ml-2 opacity-8"
    icon={faQuestionCircle}
    id="Tooltip-1"
  />
  <CounterInput
    value={currEventInfo.weeks || 0}
    onChange={(e) =>
      counterHandler(e, "weeks", currEventInfo, setCurrEventInfo)
    }
    disabled={disabledInput}
    placeholder=""
  />
  <UncontrolledTooltip placement="top" target="Tooltip-1">
    Event repeats every __ weeks
  </UncontrolledTooltip>
</div>
```